try {
    let c = a + b
} catch (error) {
    console.log("algo deu erro" + error)
}